app/controllers/ControllerHelper.scala

```diff
@@ -62,4 +62,15 @@ trait ControllerHelper extends FrontendBaseController with I18nSupport {
     }
   }
 
+  def updateDatabaseWithoutRedirect(updatedAnswers: UserAnswers, page: Page)
```

Comment:
if we are adding this in, Im guessing we need to use it on the other off ramp controllers? also is this function tested